uai2017/process/pc_responseInvitationProcess_uai2017.js

```diff
@@ -0,0 +1,34 @@
+function() {
+
+  var or3client = lib.or3client;
+  var hashKey = or3client.createHash(note.content.email, "2810398440804348173");
+  if(hashKey == note.content.key) {
+    if (note.content.response == 'Yes') {
+      console.log("Invitation replied Yes")
+      //if a user is in the declined group, remove them from that group and add them to the reviewers group
+      or3client.removeGroupMember('auai.org/UAI/2017/Program_Committee/declined', note.content.email, token)
+      or3client.addGroupMember('auai.org/UAI/2017/Program_Committee', note.content.email, token)
+      .then(function(result) {
+        var message = "The user " + note.content.email + " has accepted the invitation to serve as a reviewer."
+        //return or3client.or3request(or3client.mailUrl, { groups: [invitation.signatures[0]], subject: "OpenReview invitation accepted" , message: message}, 'POST', token);
+      })
+      .then(result => done())
+      .catch(error => console.log(error));
```

Comment:
I mean when there is an error. You need to send the error to the main process to know what happened.